### DISCUSSION TITLE
Pin to taskbar

### DISCUSSION THREAD

**Discussion by pwpeqoiq** [doublecmd/doublecmd#1797] [Suggestions]
Is it possible to add an item to the context menu: Pin to taskbar?
Возможно ли добавить пункт в контекстное меню: Закрепить на панели задачи?

https://github.com/user-attachments/assets/dbcd7d45-9c0f-4b56-a6b5-09a17aa6e558

**Comment by alexx2000** [comment]
No, see https://www.ghisler.ch/board/viewtopic.php?p=433011
